setup.py

```diff
@@ -3,23 +3,18 @@
 from io import open
 import os
 import sys
-
+from setuptools import Extension, setup
+from Cython.Build import cythonize
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 # Get the long description from the README file
 with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-cmake_args = []
-for option in ("CYTHON_LINETRACE", "CYTHON_ANNOTATE"):
-    value = os.environ.get(option)
-    if value is not None and bool(int(value)):
-        cmake_args.append("-D{}:BOOL=TRUE".format(option))
-
-# On Mac, require OSX >= 10.9 so we can use te new libc++ targeting C++11
-if sys.platform == "darwin":
-    cmake_args.append("-DCMAKE_OSX_DEPLOYMENT_TARGET:STRING=10.9")
```

Comment:
Wondering why this has anything about libc++ as harfbuzz won't use it anyway, of course we should make sure older compilers are using C++11 build flag but this isn't about it.

Reply:
I am not sure actually. We can try to disable that and see if it works.

Reply:
it targets C++11 anyway now.